tst/testinstall.tst

```diff
@@ -1412,6 +1412,25 @@ gap> M := MaximalSubsemigroups(T);;
 gap> T in M;
 false
 
+# Test for Issue 217 (bug in \in for L-class of inverse semigroups)
+gap> S := InverseMonoid(PartialPerm([2, 3, 4, 5, 6], [1, 2, 3, 4, 5]),
+> PartialPerm([1, 2, 3, 4, 5], [1, 2, 3, 4, 6]),
+> PartialPerm([1, 2, 3, 4, 6], [1, 2, 3, 5, 6]),
+> PartialPerm([1, 2, 3, 5, 6], [1, 2, 4, 5, 6]),
+> PartialPerm([1, 2, 4, 5, 6], [1, 3, 4, 5, 6]),
+> PartialPerm([1, 3, 4, 5, 6], [2, 3, 4, 5, 6]),
+> PartialPerm([1, 2, 3, 4, 5, 6], [6, 5, 4, 3, 2, 1]));;
+gap> NrMaximalSubsemigroups(S);
+11
+gap> D := DClasses(S)[2];;
```

Comment:
The ordering of the D-classes varies depending on whether `S` is an acting semigroup or a generic semigroup. So `y` should be defined before this line, and then this line should be changed to `D := DClass(S, y);`. Currently the tests are failing for this reason.